src/applications/personalization/profile-2/components/ProfileHeader.jsx

```diff
@@ -32,7 +34,7 @@ const ProfileHeader = ({
           <h2 className="vads-u-font-size--h3">{fullName}</h2>
           {latestBranchOfService && (
             <h3 className="vads-u-font-family--sans vads-u-font-size--base vads-u-font-weight--normal">
-              United States {latestBranchOfService}
+              {getServiceBranchDisplayName(latestBranchOfService)}
```

Comment:
Switched to using a new helper function to add `United States` before the branch name. the helper is smart enough to only add `United States` in some situations. For example, if the `latestBranchOfService` is `Other` we don't want to render `United States Other`